documentation/Installation.md

```diff
@@ -4661,10 +4661,10 @@ Specify the following parameters:
 
 The `chart_path` parameter specifies the absolute path on local host to the Helm chart. The URL link to chart archive is also supported.
 
-The `values_file` parameter specifies the absolute path on local host to the file with YAML formatted values for the chart that override values from `values.yaml` file from the provided chart.
+The `values_file` parameter specifies the absolute path on the local host to the file with YAML formatted values for the chart that overrides values from the `values.yaml` file from the provided chart.
```

Comment:
values ... override ... values, so probably it was initially correct? Please also update kubemarine\resources\schemas\definitions\plugins\installation\helm.json correspondingly.

Reply:
Oh, that's right. I will make it "override" like initially. I assumed it as parameter...overrides...values. 
Will also review the json file and inform you.

Thanks!

Reply:
Hi, @ilia1243,

The helm.json file is reviewed.